Networking/NetworkingTests/Mapper/OrderMapperTests.swift

```diff
@@ -325,6 +337,12 @@ private extension OrderMapperTests {
         return mapOrder(from: "order-with-line-item-attributes")
     }
 
+    /// Returns the OrderMapper output upon receiving `order-with-faulty-attributes`
+    ///
+    func mapLoadOrderWithFaultyAttributesResponse() -> Order? {
```

Comment:
Please consider mentioning in a comment that the `_measurement_data` value having object format (and not a string) is what is "faulty" here. I had to fire up the debugger to understand what in the order was the problem.